.github/workflows/integration-tests.yml

```diff
@@ -163,7 +167,7 @@ jobs:
       - name: Expected version - Start Grafana
         if: steps.has-integration-tests.outputs.DIR == 'true' && steps.should-run-expected-latest-tests.outcome == 'success'
         run: |
-          docker run -d -p 3000:3000 --add-host=host.docker.internal:host-gateway --name $PLUGIN_ID -v ${PWD}/dist:/var/lib/grafana/plugins/$PLUGIN_ID -v ${PWD}/provisioning:/etc/grafana/provisioning -e GF_DEFAULT_APP_MODE -e GF_INSTALL_PLUGINS -e GF_AUTH_ANONYMOUS_ORG_ROLE -e GF_AUTH_ANONYMOUS_ENABLED -e GF_AUTH_BASIC_ENABLED grafana/grafana:$EXPECTED_GRAFANA_VERSION
+          docker run -d -p 3000:3000 --add-host=host.docker.internal:host-gateway --name $PLUGIN_ID -v ${PWD}/dist:/var/lib/grafana/plugins/$PLUGIN_ID -v ${PWD}/provisioning:/etc/grafana/provisioning -e GF_DEFAULT_APP_MODE -e GF_INSTALL_PLUGINS -e GF_AUTH_ANONYMOUS_ORG_ROLE -e GF_AUTH_ANONYMOUS_ENABLED -e GF_AUTH_BASIC_ENABLED ${GITHUB_ENV:+-e GF_FEATURE_TOGGLES_ENABLE=${{ env.GF_FEATURE_TOGGLES_ENABLE }}} grafana/grafana:$EXPECTED_GRAFANA_VERSION
```

Comment:
Think we can simplify this with docker compose. Try this: 
```
- name: Start Grafana
        run: |
          docker-compose pull
          docker-compose up -d
```

Reply:
I am not sure we can do it. we are running 3 different versions of grafana here: current, latest and canary. We can change it, but I prefer to make the separate task for it, if you do not mind.

Reply:
You'd just pass grafana version as an argument `GRAFANA_VERSION=$EXPECTED_GRAFANA_VERSION`. Then you wouldn't need that env file. 

this is not a blocker from my side. leaving it up to you